src/main/java/duke/templar/TaskManager.java

```diff
@@ -0,0 +1,226 @@
+package duke.templar;
+
+import duke.exception.*;
+import java.util.ArrayList;
+
+
+public class TaskManager {
+    public static final String MESSAGE_DIVIDER = "____________________________________________________________";
+    boolean valid;
+    public final ArrayList<Task> tasks;
+    int[] taskDone = new int[100]; //this array stores 1 or 0 - task done or not
+    String[] validCommands = new String[] {
+        "print commands",
+        "todo" ,
+        "deadline",
+        "event" ,
+        "list" ,
+        "done" ,
+        "delete",
+        "bye"
+    };
+
+
+    public TaskManager(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /*
+    * method processes the input to determine which class of task it is and then create the correct object accordingly
```

Comment:
For java doc comments, there is no need to start with `method processes...` can simply describe it as `Processes the input...`